api/http/handler/settings/settings_update.go

```diff
@@ -103,6 +103,11 @@ func (handler *Handler) settingsUpdate(w http.ResponseWriter, r *http.Request) *
 		return &httperror.HandlerError{http.StatusInternalServerError, "Unable to retrieve the settings from the database", err}
 	}
 
+	if handler.demoService.IsDemo() {
+		*payload.EnableTelemetry = false
+		*payload.LogoURL = ""
```

Comment:
should it be a part of `initDemoLocalEndpoint()` as well?

Reply:
it is done in https://github.com/portainer/portainer/blob/efddcd68e10525a66ce7531efa986c86217c8b0c/api/demo/init.go#L65 but I guess this is done here to prevent overwriting the demo settings

Reply:
sorry wasn't clear.
I can see us setting `EnableTelemetry = false` but not the `LogoURL`: https://github.com/portainer/portainer/blob/efddcd68e10525a66ce7531efa986c86217c8b0c/api/demo/init.go#L62
do we want to set it as well as a part of init?

Reply:
well, I guess it's not set as empty because that's the default value. but I'll add it to make it clear.

Reply:
ahh, missed that fact 👍